run/logging-manual/e2e_test.py

```diff
@@ -136,4 +136,6 @@ def test_end_to_end(services):
             # If there are any results, exit loop
             break
 
+        time.sleep(3 * x)  # Variable backoff
```

Comment:
- A more common term is `Linear backoff`
- Comment could probably be on preceding line?